msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IMDSManagedIdentity.java

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class IMDSManagedIdentity extends AbstractManagedIdentitySource{
```

Comment:
1. Not public
2. nit: Naming - `ImdsManagedIdentitySource` to be consistent with .NET?

Reply:
Making it package private.
Missed the naming. Thanks for pointing it out.